i18n-common.yml

```
@@ -351,6 +351,10 @@
   data:
     EN: " to see how to contact to the lazy author!"
     ZH_HANS: 查看如何联系作者这个懒虫！
+- id: UnfinishedText2
+  data:
+    EN: "Waiting for development please..."
```

```suggestion
    EN: "Coming Soon~"
```

这明显是直接按照中文一个字一个字翻译过去的。。。